services/121-service/src/transaction-job-processors/transaction-job-processors.service.ts

```diff
@@ -207,6 +207,7 @@ export class TransactionJobProcessorsService {
       amountTransferred:
         intersolveVisaDoTransferOrIssueCardReturnDto.amountTransferredInMajorUnit,
       bulkSize: input.bulkSize,
+      userId: input.userId,
```

Comment:
So you added the userid when sending a message after a successful transfer and after re-issuing a card, right? What was the bug? That the username did not show up as sender of the message? Or that sending messages did not work at all?